examples/userapi/05_conditional_dimension.ipynb

```diff
@@ -0,0 +1,853 @@
+{
```

Comment:
<p>shouldn't it be </p><p> </p><pre class="ql-syntax" spellcheck="false">from devito import Gt
</pre><p>ie</p><ul><li>no need to import all other relations</li><li>should not import from an internal (.types) but rather directly from devito</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
> I can only import from .types

 then this must be fixed right...? why should a user import from types?